homework_4.py

```diff
@@ -0,0 +1,57 @@
+test_info_1 = "Please, write your name: "
+test_info_2 = "Write the letters you want to delete from your name?: "
+test_info_3 = "Your new name is: "
+answer_info_1 = input(f"{test_info_1}")
+answer_info_2 = input(f"{test_info_2}")
+answer_info_3 = answer_info_1.replace(answer_info_2, "")
+result_info = input(f"{test_info_3} {answer_info_3}")
+
+
+example_str = input("Please, enter your sentence: ")
+title_str = example_str.title()
+print(title_str)
+
+
+regular_str = 'Python'
+upend_str = regular_str[::-1]
+print(upend_str)
+
+
+entrance_str_1 = input("Please, enter your first comment: ")
+entrance_str_2 = input("Please, enter your second comment: ")
+if len(entrance_str_1) < len(entrance_str_2):
+    print("First comment is short")
+else:
+    print("First comment is long")
+
```

Comment:
len() only check length for string(count of symbols). This is invalid decision. Because test and aaaa have the same len.

You can use
str1 = input("Enter first string: ")
str2 = input("Enter second string: ")
result = str1 == str2

print(f'String {str1} and string {str2} have equals result {result}')

Reply:
fixed